### PR TITLE
add: deprecate flags

### PR DESCRIPTION
**Pull request by dberenbaum**
3.0 deprecations for `add/import/import-url`

### CODE REVIEW

File: content/docs/command-reference/add.md [hunk 1]
```diff
@@ -6,16 +6,26 @@ file.
...
 
 ```usage
-usage: dvc add [-h] [-q | -v] [-R] [--no-commit] [--external]
-               [--glob] [--file <filename>] [-o <path>]
-               [--to-remote] [-r <name>] [-j <number>] [-f]
+usage: dvc add [-h] [-q | -v] [-f] [--no-commit] [--external]
+               [--glob] [-o <path>]
+               [--to-remote] [-r <name>] [--remote-jobs <number>]
                [--desc <text>] [--meta key=value] [--label <str>]
                [--type <str>]
```
Comment (skshetry) [inline content/docs/command-reference/add.md:13]:
```suggestion
```

These options were also removed in 3.0. @dberenbaum, do you mind removing these in the same PR?

File: content/docs/command-reference/add.md [hunk 2]
```diff
@@ -6,16 +6,26 @@ file.
...
                targets [targets ...]
 positional arguments:
   targets               Files or directories to add
 ```
 
+<details>
+
+### Options deprecated in 3.0
```
Comment (skshetry) [inline content/docs/command-reference/add.md:21]:
```suggestion
### Options removed in 3.0
```

deprecate = discouraged to use but still supported.
These were removed in 3.0 without a replacement.

Comment (skshetry) [reply content/docs/command-reference/add.md:21]: I am not sure if we need to list these, as they don't exist anywhere else in the docs (like what it did in 2.0, etc).

And we don't support multiple versions in docs. :)

File: content/docs/command-reference/add.md (hunk 1 above)
Comment (dberenbaum) [reply content/docs/command-reference/add.md]: Added, PTAL

File: content/docs/command-reference/add.md (hunk 2 above)
Comment (dberenbaum) [reply content/docs/command-reference/add.md]: I see we are already violating this in other PRs and it seems there's a lot of pushback, so I'll drop for now. I guess we could always come back and add these notices if we see confusion or people complain.

**Review by skshetry — APPROVED** [review]

**Pull request merged**